app/main.py

```diff
@@ -2,0 +1,26 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split(" ")
+    mv = command[0]
+    file_to_move = command[1]
+    destination = command[2].split("/")
+    if mv == "mv" and file_to_move != "".join(destination):
+        try:
+            with open(file_to_move, "r") as f:
+                data = f.read()
+            os.remove(file_to_move)
+        except FileNotFoundError:
+            print("File not found")
```

Comment:
Clean Code issue: You are using a print statement to inform the user that the file was not found. Although this is not wrong, it's better to use a logger to log this kind of information. This way, you can control the level of logging and also redirect the logs to a file if needed.